ui_tests/src/test/resources/com/saucedemo/account/SignInPositive.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Sign In
+  @signIn
+  Scenario: Sign In with valid username and password
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+  @signIn
+  Scenario: Sign In with valid username and password
+    Given I open start page
+    When I set 'problem_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+  @signIn
+  Scenario: Sign In with valid username and password
+    Given I open start page
+    When I set 'performance_glitch_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+  @signIn
+  Scenario: Sign In with valid username and password
+    Given I open start page
+    When I set 'locked_out_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then Error message 'Epic sadface: this user name has been locked out ' is available on Sign in page
```

Comment:
@Firdavs11 This is not a positive, but negative test case. As we are verifying that not entitled user cannot sign in.
Could you please rename this scenario and move it to signInNegative.feature file